### PR TITLE
Update AWS SDK, CURL and Protobuf

### PR DESCRIPTION
**Pull request by mzient**
AWS-SDK: 1.11.418 -> 1.11.458
CURL: 8.10.1 -> 8.11.0
protobuf: v28.2 -> v29.0

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,19 +42,19 @@ The repository consists mostly of externally hosted subrepositories:
...
 +------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
-| `curl <https://github.com/curl/curl.git>`_                                   | `8.10.1 <https://github.com/curl/curl/releases/tag/curl-8_10_1>`_                                                 | `CURL license <https://github.com/curl/curl/blob/master/LICENSES/curl.txt>`_                                        |
+| `curl <https://github.com/curl/curl.git>`_                                   | `8.10.1 <https://github.com/curl/curl/releases/tag/curl-8_11_0>`_                                                 | `CURL license <https://github.com/curl/curl/blob/master/LICENSES/curl.txt>`_                                        |
```
Comment (JanuszL) [inline README.rst:53]:
```suggestion
| `curl <https://github.com/curl/curl.git>`_                                   | `8.11.0 <https://github.com/curl/curl/releases/tag/curl-8_11_0>`_                                                 | `CURL license <https://github.com/curl/curl/blob/master/LICENSES/curl.txt>`_                                        |
```

File: README.rst [hunk 2]
```diff
@@ -42,19 +42,19 @@ The repository consists mostly of externally hosted subrepositories:
...
 +------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
-| `aws_sdk_cpp <https://github.com/aws/aws-sdk-cpp.git>`_                      | `1.11.418 <https://github.com/aws/aws-sdk-cpp/tree/1.11.418>`_                                                    | `Apache 2.0 license <https://github.com/aws/aws-sdk-cpp/blob/main/LICENSE.txt>`_                                    |
+| `aws_sdk_cpp <https://github.com/aws/aws-sdk-cpp.git>`_                      | `1.11.418 <https://github.com/aws/aws-sdk-cpp/tree/1.11.458>`_                                                    | `Apache 2.0 license <https://github.com/aws/aws-sdk-cpp/blob/main/LICENSE.txt>`_                                    |
```
Comment (JanuszL) [inline README.rst:57]:
```suggestion
| `aws_sdk_cpp <https://github.com/aws/aws-sdk-cpp.git>`_                      | `1.11.458 <https://github.com/aws/aws-sdk-cpp/tree/1.11.458>`_                                                    | `Apache 2.0 license <https://github.com/aws/aws-sdk-cpp/blob/main/LICENSE.txt>`_                                    |
```

**Review by JanuszL — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (mzient) [reply README.rst]: Fixed.

File: README.rst (hunk 2 above)
Comment (mzient) [reply README.rst]: Fixed.

**Review by stiepan — APPROVED** [review]

**Pull request merged**